article/app/controllers/LiveBlogController.scala

```diff
@@ -98,7 +105,12 @@ class LiveBlogController(
     }
   }
 
-  private[this] def renderWithRange(path: String, range: BlockRange, filterKeyEvents: Boolean)(implicit
+  private[this] def renderWithRange(
+      path: String,
+      range: BlockRange,
+      filterKeyEvents: Boolean,
+      topMentionResult: Option[TopMentionsResult],
```

Comment:
I don't think this is used?

Reply:
Yea that's right, I added it here because it'd be needed in this method in upcoming PRs where we work on the pagination model. Is that OK to leave it for now?

Reply:
sure!